ketcher-autotests/tests/User-Interface/Floating windows/floating-windows.spec.ts

```diff
@@ -0,0 +1,166 @@
+import { test } from '@playwright/test';
+import {
+  selectTopPanelButton,
+  TopPanelButton,
+  takeEditorScreenshot,
+  pasteFromClipboardAndAddToCanvas,
+  openFileAndAddToCanvas,
+  openFile,
+  FILE_TEST_DATA,
+  pasteFromClipboard,
+  waitForLoad,
+  pressButton,
+} from '@utils';
+
+test.describe('Click and drag FG on canvas', () => {
+  test.beforeEach(async ({ page }) => {
+    await page.goto('');
+  });
+
+  test.afterEach(async ({ page }) => {
+    // await resetCurrentTool(page);
+    await takeEditorScreenshot(page);
+  });
+
+  test('Calculate values (data on canvas)', async ({ page }) => {
+    /*
+      Test case: EPMLSOPKET-3998
+      Description: verify the floating window with calculated values 
+    */
+    await openFileAndAddToCanvas('bicycle.mol', page);
+    await selectTopPanelButton(TopPanelButton.Calculated, page);
+  });
+
+  test('Calculated values: check accuracy', async ({ page }) => {
+    /*
+      Test case: EPMLSOPKET-3999(1)
+      Description: verify 0 decimal places after the dot for calculated values 
+    */
+    await openFileAndAddToCanvas('bicycle.mol', page);
+    await selectTopPanelButton(TopPanelButton.Calculated, page);
+    await page
```

Comment:
This bunch of action is also not very clear for me. Could you add testId for item you want to click? Or create a function for this?

Reply:
> This bunch of action is also not very clear for me. Could you add testId for item you want to click? Or create a function for this?

Not hand made. Should I remove that?